README.md

```diff
@@ -1,3 +1,43 @@
-# Membership
+# Node.js Foundation - Membership
 
-Repo to work out the system for individual membership in the foundation.
+## What types of members are there?
+
+* Enterprise
+* Individual
+* Student
+* Active Contributor
+
+## What are the conditions of eligibility?
+
+TBD - Join the discussion on this at [issues/8](https://github.com/nodejs/membership/issues/8)
+
+## How to join
```

Comment:
There's actually just three "classes" of membership: corporate, associate and individual.

Student, Active Contributor, etc are all "Individual Members."

I would talk about the difference between corporate or individual, ignore associate (which is reserved for universities and non-profits, we don't actually have any of these yet).